zerver/tests/test_docs.py

```diff
@@ -406,6 +406,26 @@ def test_help_settings_links(self) -> None:
         self.assertIn("<strong>Preferences</strong>", str(result.content))
         self.assertNotIn("/#settings", str(result.content))
 
+    def test_help_users_tab_links(self) -> None:
+        result = self.client_get("/help/invite-new-users")
+        self.assertEqual(result.status_code, 200)
+        # The full sentence hasn't been tested since there's a bunch of
+        # whitespace in the HTML after the invitations link and we
+        # don't want that whitespace to be part of the test since it
+        # might change in the future.
```

Comment:
I'm not sure whether I should test the full sentence or not because of the whitespace. I saw that the other tests are also not testing the full sentences and decided to do something similar here. See the whitespace after invitations link in the code block below

```
Navigate to the <a href="/#organization/users/invitations">Invitations</a>                     tab of the <strong>Organization settings</strong> menu.
```